content/how-to-deploy-supertokens-with-react-nodejs-express-on-vercel/index.md

```diff
@@ -0,0 +1,152 @@
+* **Simple data model:** Supertokens has a modular structure that allows you to add database tables with fewer configurations.
+* **Priced for startups:** Supertokend has a generous limit and pricing for our managed service, and it's free forever for self-hosted.
+* **Integrate with any service:** You can set up your content delivery service (e.g., SMS or email) for free.
+
+
+## What is Vercel
+
+Vercel is a platform for frontend frameworks and static sites that can be integrated with your headless content, commerce, or database. It offers a frictionless developer experience to handle the difficult tasks of deploying instantaneously, scaling automatically, and presenting tailored content globally. It's built for the frontend experience; create hybrid frontends using tools like Next.js, create lightweight event-driven APIs, and deploy to our Global Edge Network.
+
+
+## Getting started
+
+To get started with this tutorial, first set up an **email password + social login with express and react** on the local computer. For that, see this [blog](https://supertokens.com/blog/how-to-set-up-social-and-email-password-login-with-reactjs). After creating and testing your application on your local computer, follow the steps below to set up the application for deployment on vercel.
+
+
+
+1. Create an **api** folder in the frontend (**my-demo-app**) part of the application.
+2. Copy the folders and files in the backend part of the application to the **api** folder.
+3. Rename the **server.js** file in the backend to **index.js**
+4. Open the **index.js** and export **app** at the end of the file as shown below
+
+    ```js
+    module.exports = app;
+
+    ```
+
+
+> Note: We are deploying our backend as a standalone express app, not with the default **/pages/api** serverless functions that Vercel implements.
+
+You can find an example repository for SuperTokens + emailpassword + Vercel auth [on our Github](https://github.com/supertokens/supertokens-auth-react/tree/master/examples/with-emailpassword-vercel).
+
+
+## Test application locally
+
+In order to run the application locally, you need to make sure that the websiteDomain and apiDomain values on the frontend and backend point to localhost. In our [example app](https://github.com/supertokens/supertokens-auth-react/tree/master/examples/with-emailpassword-vercel), we make the apiDomain [http://localhost:3001](http://localhost:3001) and websiteDomain http://localhost:3000.
+
+
+## When deploying to vercel
+
+To enable project run in production and the inspect sites for vercel, you need to make the websiteDomain and appDomain point to the URLs generated by vercel. The production URL is always the same for an app, but the inspect URL keeps changing. To allow it to work in this dynamic condition, we must set the values of apiDomain and websiteDomain dynamically.
```

Comment:
Make sure all vairables like apiDomain and websiteDomain that are being referred to are in inline code snippets with a ``